purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesOrchestrator.kt

```diff
@@ -1333,7 +1386,32 @@ internal class PurchasesOrchestrator(
         )
     }
 
-    // endregion
+    private fun trackPurchaseStarted(productId: String, productType: ProductType) {
```

Comment:
Not a huge deal but should we  add the `IfNeeded` suffix to these functions? I see that wasn't added to the functions above, but we've been adding it in other places... I think it might be worth it but NABD.

Reply:
Done in https://github.com/RevenueCat/purchases-android/pull/2291/commits/4816e6619d3f1f112590beefb5c963ee860ad86a